cli/src/klio_cli/commands/job/gke.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2019-2020 Spotify AB
```

Comment:
It's 2021 :)

Also, all the `GKE` naming (even `runner: DirectGKERuner` in the job config) should probably be renamed to not make it sound like GKE is needed, as I would assume running a pipeline could work on AWS's EKS and elsewhere. What do y'all think?